content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -86,14 +86,17 @@ In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
 ## Drop-in directory for kubelet configuration files {#kubelet-conf-d}
 
-As of Kubernetes v1.28.0, the kubelet has been extended to support a drop-in configuration directory. The location of it can be specified with
-`--config-dir` flag, and it defaults to `""`, or disabled, by default.
+{{<feature-state for_k8s_version="v1.28" state="alpha" >}}
 
-You can only set `--config-dir` if you set the environment variable `KUBELET_CONFIG_DROPIN_DIR_ALPHA` for the kubelet process (the value of that variable does not matter).
-For Kubernetes v{{< skew currentVersion >}}, the kubelet returns an error if you specify `--config-dir` without that variable set, and startup fails.
-You cannot specify the drop-in configuration directory using the kubelet configuration file; only the CLI argument `--config-dir` can set it.
+You can specify a drop-in configuration directory for the kubelet. By default, the kubelet does not look
+for drop in configuration files anywhere; you must specify a path.
+For example: `--config-dir=/etc/kubernetes/kubelet.conf.d`
 
-One can use the kubelet configuration directory in a similar way to the kubelet config file.
+For Kubernetes v1.28 to v{{< skew currentVersion >}}, you  can only specify `--config-dir` if you also set
```

Comment:
wouldn't this be `v1.28 to v1.29` (instead of currentVersion, which would be 1.30)

Reply:
I followed [this](https://github.com/kubernetes/website/pull/44985#discussion_r1475839135) comment

Reply:
The current version of Kubernetes is v1.29. 

> I've suggested changes that I think are appropriate for a PR targeting _main_; it will render right. Then, once this is merged to _main_ and has been onwards merged to _dev-1.30_, send in a second PR to promote the feature to beta. The second PR should have _dev-1.30_ as its base branch.

The approach I've suggest would work even if the k/k update to kubelet drop-in config didn't land until a later minor release.

Reply:
@sftim does the latest push make sense?

Reply:
Yep